src/pivot-table/hooks/use-column-width.ts

```diff
@@ -49,14 +79,26 @@ export default function useColumnWidth(
         return pseudoDimensionWidth / rect.width;
       }
 
-      const { qFallbackTitle, qApprMaxGlyphCount } = qDimensionInfo[dimIndex];
-      const hasChildNodes = index < qNoOfLeftDims - 1; // -1 as the last column can not be expanded or collapsed
-      const collapseExpandIconSize = hasChildNodes ? EXPAND_ICON_WIDTH : 0;
+      const { qFallbackTitle, qApprMaxGlyphCount, columnWidth } = qDimensionInfo[dimIndex];
+      // percentage
+      if (columnWidth?.type === "pixels") {
+        return (columnWidth.pixels || ColumnWidthValues.PixelsDefault) / rect.width;
+      }
+      // pixels
+      if (columnWidth?.type === "percentage") {
+        // TODO: do we want the percentage value to represent the entire chart or just the 75% that you can occupy with the left side
```

Comment:
In the future when the left grid can be scrolled horizontally by itself, as in not scrolling the top or measure grids. 100% could mean the entire chart, but the actual visible width of the left grid would be something like 75% of the entire chart. So the user could scroll the remaining 25% in the left grid.

But until then I think it's better have "percentage" value be based on the actual width of the left grid (75% of the entire chart).